spec/workers/flickr_photos_importer_spec.rb

```diff
@@ -27,23 +32,44 @@
         allow(batch1_photos).to receive(:pages).and_return(3)
         allow(batch2_photos).to receive(:pages).and_return(3)
         allow(batch3_photos).to receive(:pages).and_return(3)
-        allow(importer).to receive(:get_photos).with('flickr id', 'user', per_page: FlickrPhotosImporter::MAX_PHOTOS_PER_REQUEST, extras: FlickrPhotosImporter::EXTRA_FIELDS, page: 1).and_return(batch1_photos)
-        allow(importer).to receive(:get_photos).with('flickr id', 'user', per_page: FlickrPhotosImporter::MAX_PHOTOS_PER_REQUEST, extras: FlickrPhotosImporter::EXTRA_FIELDS, page: 2).and_return(batch2_photos)
-        allow(importer).to receive(:get_photos).with('flickr id', 'user', per_page: FlickrPhotosImporter::MAX_PHOTOS_PER_REQUEST, extras: FlickrPhotosImporter::EXTRA_FIELDS, page: 3).and_return(batch3_photos)
+        allow(importer).to receive(:get_photos).with(
```

Comment:
I started DRY-ing up these importer args, but it got tricky and ugly with the varying values in the options. I decided to clean up the long lines for now but leave the refactoring for another time.